requirements.txt

```diff
@@ -1,3 +1,4 @@
 urllib3==1.25.9
 requests==2.24.0
 six==1.15.0
+python-dateutil==2.8.1
```

Comment:
Tests failed locally when `pip install -r requirements-dev.txt` since it's missing this as dependency. Is that the intention or just an overlook?

Reply:
I think readme needs to be updated that both requirements.txt and requirements-dev.txt need to be installed with pip if tests need to be ran.

Reply:
For some reason now the tests fail even with both installs. Even two `py.test test/` fails to `E   ImportError: cannot import name 'ZONE'`

tox has 5 failing tests to `E   ModuleNotFoundError: No module named 'dateutil'`

Reply:
Strange that tests on travis pass but not locally. I will double check locally, but I remember it passing on my env.

Reply:
Yeah I think at one point they passed for me too. Would need to be able to recreate that env where they pass though. I have no idea how to fix these though 😅

Reply:
seems that the issues I had with installs were MacOs Big Sur related.